packages/tasks/src/model-libraries.ts

```diff
@@ -140,18 +134,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://aphp.github.io/edsnlp/latest/",
 		filter: false,
 		snippets: snippets.edsnlp,
-		countDownloads: {
-			wildcard: { path: "*/config.cfg" },
-		},
+		countDownloads: `path_filename:"config.cfg" AND _exists_:"path_prefix"`,
```

Comment:
```suggestion
		countDownloads: `path_filename:"config.cfg"`,
```

i would guess we probably don't care about this part i.e. if the file is at the root, we can count it

Reply:
I prefer have at least one example and how we can achieve this

Reply:
sure, ok to keep one